sui_types/src/committee.rs

```diff
@@ -68,4 +70,26 @@ impl Committee {
         // then (N + 2) / 3 = f + 1 + k/3 = f + 1
         (self.total_votes + 2) / 3
     }
+
+    /// Given a sequence of (AuthorityName, value) for values that are ordered, provide the
+    /// value at the particular threshold by stake. This orders all provided values and pick
+    /// the appropriate value that has under it threshold stake. You may use the function 
+    /// `quorum_threshold` or `validity_threshold` to pick the f+1 or 2f+1 thresholds 
+    /// respectivelly.
+    pub fn robust_value<A, V>(&self, items: impl Iterator<Item=(A, V)>, threshold: usize) -> (AuthorityName, V) 
+        where A : Borrow<AuthorityName> + Ord,
+              V : Ord,
+    {
+        debug_assert!(threshold < self.total_votes);
+
+        let vec : Vec<_> = items.map(|(a, v)| (v, self.voting_rights[a.borrow()], *a.borrow())).sorted().collect();
```

Comment:
No need to call `collect` since we will be iterating below

Reply:
Will there be a difference if we sort it in descending order instead of ascending?